evm/constants.py

```diff
@@ -1,10 +1,16 @@
+import logging
+
 from eth_typing import (
     Address,
     Hash32
 )
 from eth_utils import denoms
 
 
+# FIXME: Need to move this somewhere else and possibly provide a custom Logger instance with a
+# method (e.g. .trace()) that uses this level.
+LOG_TRACE = logging.DEBUG - 1
```

Comment:
We do have a `TraceLogger` that defines log level `5` for tracing

https://github.com/ethereum/py-evm/blob/6708b9eedde39b3b8d4c0db57a7bd2b1770c72d9/evm/utils/logging.py#L8-L14

So, I think we should be able to just use `logger.trace(...)` where needed. In fact, the `computation` is already doing that.

https://github.com/ethereum/py-evm/blob/6708b9eedde39b3b8d4c0db57a7bd2b1770c72d9/evm/vm/computation.py#L543-L548